Peer/Apps/AppBuilder/HandlerWrapper.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Peer.Apps.AppBuilder;
+
+public class HandlerWrapper<T> : IHandler
```

Comment:
Could this just be an extension method on `IHandler`?

Reply:
This doesn't implement IHandler<T>

Reply:
Dunno if that's intentional or not

Reply:
Alternatively could just force IHandler<T> to implement the object method but tbh that feels more gross